platform/android/java/lib/build.gradle

```diff
@@ -2,6 +2,9 @@ apply plugin: 'com.android.library'
 
 dependencies {
     implementation "com.android.support:support-core-utils:28.0.0"
+
+    //ARCore specific, needs to become options
+    implementation "com.google.ar:core:1.10.0"
```

Comment:
Indeed, I have no sound ideas here which is why the remark is in the code.

Reply:
@akien-mga we could use your input here. I did originally put all the code in a module so it could in theory be turned off. I'm not sure how we would then exclude these dependencies from the build but it would allow compiling Godot with and without ARCore support.